services/web/client/source/class/qxapp/components/widgets/FileManager.js

```diff
@@ -46,14 +34,14 @@ qx.Class.define("qxapp.components.workbench.widgets.FileManager", {
     }, this);
 
     let tree = this.__mainTree = new qx.ui.tree.Tree();
-    this.add(tree, {
+    this._add(tree, {
```

Comment:
since the filemanager is a widget now, I would expect the childControl concept to be used
http://www.qooxdoo.org/devel/pages/desktop/ui_develop.html